implementations/prometheus/e2e_stats_test.go

```diff
@@ -51,15 +52,15 @@ const alloyMetadataRetried = "alloy_queue_metadata_network_retried_total"
 
 const alloyNetworkTimestamp = "alloy_queue_series_network_timestamp_seconds"
 
-const alloyDrift = "alloy_queue_series_timestamp_drift_seconds"
+//const alloyDrift = "alloy_queue_series_timestamp_drift_seconds"
 
 const alloyMin = "alloy_queue_series_parallelism_min"
 const alloyMax = "alloy_queue_series_parallelism_max"
 const alloyDesired = "alloy_queue_series_parallelism_desired"
 
-const alloyMetaMin = "alloy_queue_metadata_parallelism_min"
-const alloyMetaMax = "alloy_queue_metadata_parallelism_max"
-const alloyMetaDesired = "alloy_queue_metadata_parallelism_desired"
+// const alloyMetaMin = "alloy_queue_metadata_parallelism_min"
```

Comment:
These were flagged as unused, should they be?

Reply:
Theoretically they could be used, but currently in practice we dont support metadata.